woocommerce/woocommerce-ios
# Removed ShimmerModifier class and included SwiftUI-Shimmer using SPM

Closes #4248 

## Description
As discussed [here](https://github.com/woocommerce/woocommerce-ios/pull/4229#discussion_r636264165) it's better to include `SwiftUI-Shimmer` as a separate package using Swift Package Manager, which will include also the MIT License and also informs us that “this is third-party code and we shouldn't touch it”. In this PR, I removed our existing class `ShimmerModifier` and I included [SwiftUI-Shimmer](https://github.com/markiv/SwiftUI-Shimmer) using SPM.

## Testing
1. Launch the app in debug mode (because of the feature flag).
2. Open an order detail that is eligible for the shipping label.
3. Press the button "Create Shipping Label".
4. Press the button "Continue" in Create Shipping Label form, under the **Ship From** cell. Confirm the address.
5. Press the button "Continue" in Create Shipping Label form, under the **Ship To** cell. Confirm the address.
6. Press the button "Continue" in Create Shipping Label form, under the **Package Details** cell.
7. You should see the screen that shows the Package Details screen and all the items that will be shipped.
8. Press the "Package selected" row. 
9. Select a package, then press "Done".
10. Add a Total package weight, press "Done".
11. The "Shipping Carrier and Rates" cell should be enabled now. Press it.
12. You should see the new screen "Carrier and Rates". It should display the ghost cells like before using SwiftUI-Shimmer, then the data fetched from the API.


Update release notes:
